innsender/src/main/resources/application.yml

```diff
@@ -187,16 +196,23 @@ spring:
   datasource:
     url: jdbc:postgresql://${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_DATABASE}?user=${DATABASE_USERNAME}&password=${DATABASE_PASSWORD}
     hikari:
-      minimum-idle: 1
       maximum-pool-size: 20
-      connection-timeout: 50000
-      max-lifetime: 50001
+      max-lifetime: 30000
       validationTimeout: 20000
       leak-detection-threshold: 60000
     dbcp2:
       test-on-borrow: true
       validation-query: "SELECT 1"
 
+  jpa:
+    open-in-view: false
```

Comment:
Vi bør ikke kjøre med `open-in-view: true` under lokal utvikling når det er slått av for dev og prod. Verdien bør være lik for å unngå uforståelige bugs.

Men det var kanskje ikke meningen å endre på dette i forbindelse med denne saken?